src/database/entities/cart.entity.ts

```diff
@@ -0,0 +1,29 @@
+import {Column, Entity, JoinColumn, OneToMany, PrimaryGeneratedColumn} from 'typeorm';
+import {CartItemEntity} from "./cart-item.entity";
+
+export enum STATUS {
+    OPEN = 'OPEN',
+    ORDERED = 'ORDERED'
+}
+
+@Entity({ name: 'carts'})
+export class CartEntity {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @Column({ type: 'uuid', unique: true, nullable: false })
+    user_id: string;
```

Comment:
I saw you are using some mapper functions to handle this class later. Like `orderEntityToOrder` or smth. Im not sure, but you can just use `createdAt` case, because you are using `namingStrategy: new SnakeNamingStrategy(),`. Or you can just change name in metadata of `@Column`. And then juss use this class.
![image](https://user-images.githubusercontent.com/94276879/231369430-23f356cc-f453-4bfb-b89a-abb1ecbcc6f9.png)


```
@Column({ type: 'uuid', unique: true, nullable: false, name: 'user_id' })
    userId: string;
   ```

Reply:
In fact, I was a little embarrassed by the inconsistencies in the assignment, because the requirements for the data structure in the database and in the original project are different. Therefore, in my final code, there are two types of entities - entities that are most suitable for describing the data structure in db, and data transfer objects - objects that were in the project initially. To convert one type of entities to others, I use the mapper functions. So this conversion is not related to the SnakeNamingStrategy at all.